__tests__/api.test.js

```diff
@@ -0,0 +1,59 @@
+const request = require("supertest");
+const app = require("./../app");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("./../db/data/test-data/index");
+const testData = require("./../db/data/test-data/index.js");
+
+const seed = require("./../db/seeds/seed");
+const db = require("./../db/connection");
+const jestSorted = require("jest-sorted");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET /api ", () => {
+  test("responds with a JSON object containing a 'msg' property and status code 200", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .expect("Content-Type", "application/json; charset=utf-8")
+      .then((response) => {
+        expect(response.body).toEqual({ msg: "API is running!" });
```

Comment:
Get /api is supposed to respond with JSON of the endpoints and examples of their responses, almost like instructions. it should not respond with a message like this

Reply:
should i do it in ticket 3? Because it was a separate ticket 3.5